nutanix/resource_nutanix_virtual_machine.go

```diff
@@ -917,6 +922,9 @@ func resourceNutanixVirtualMachineRead(d *schema.ResourceData, meta interface{})
 	if err := d.Set("parent_reference", flattenReferenceValues(resp.Status.Resources.ParentReference)); err != nil {
 		return fmt.Errorf("error setting parent_reference for Virtual Machine %s: %s", d.Id(), err)
 	}
+	if err := d.Set("use_hot_add", d.Get("use_hot_add")); err != nil {
+		return fmt.Errorf("error setting use_hot_add for Virtual Machine %s: %s", d.Id(), err)
```

Comment:
Why did you use d.Get() func to retrieve the use_hot_add value? Does the server not return that value?

Reply:
That's correct, this parameter is made to force reboot using logic, there are not api for that case.

Reply:
okay make sense